testament/important_packages.nim

```diff
@@ -36,7 +42,8 @@ pkg "gnuplot"
 pkg "hts", "nim c -o:htss src/hts.nim"
 pkg "illwill", "nimble examples"
 pkg "inim"
-pkg "itertools", "nim doc src/itertools.nim"
+pkg "itertools", "git checkout pr_fix_for_new_hashes2 && nim doc src/itertools.nim".toCommand, url = "https://github.com/timotheecour/itertools"
```

Comment:
![no god no](https://i.giphy.com/media/ToMjGpx9F5ktZw8qPUQ/giphy.webp)

Reply:
I've now removed hotfix now that you've merged it; it's the only way i know of to fix CI (or test a nim PR) before a hotfix is merged upstream

Reply:
> I've now removed hotfix now that you've merged i

Can't you in future just wait a bit, instead of flooding our CIs?